pkgs/development/libraries/folks/default.nix

```diff
@@ -91,11 +91,7 @@ stdenv.mkDerivation (finalAttrs: {
   doCheck = false;
 
   # Prevents e-d-s add-contacts-stress-test from timing out
-  checkPhase = ''
-    runHook preCheck
-    meson test --timeout-multiplier 4
-    runHook postCheck
-  '';
+  mesonCheckFlags = [ "--timeout-multiplier" "4" ];
```

Comment:
Could you please move the comment inside so that it does not get disconnected if more flags are added?

Reply:
Done.